src/SME.SGP.Aplicacao/Integracoes/ServicoEOL.cs

```diff
@@ -19,15 +19,16 @@ public ServicoEOL(HttpClient httpClient)
             this.httpClient = httpClient;
         }
 
-        public async Task<AlterarSenhaRespostaDto> AlterarSenha(string login, string novaSenha)
```

Comment:
essa modificação acredito que possa gerar confusão. 
Usuário fica vago, eu envio login ou código rf do usuário neste caso?
Se for um usuário externo com login ele irá poder utilizar essa funcionalidade? 
Acredito que fique melhor diferenciar se é o login ou código rf.